misc/ref/debye.py.out

```diff
@@ -1 +1,2 @@
-error 7.013244e-14
+error 7.009307e-14
+time old 0.525255, time new 0.319653
```

Comment:
I don't think anything should have changed here, and the error looks like machine precision, so there should be no reason to update these test results.